scripts/ngi_aggregate_statistics

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+""" 
+This is a script to create NGI aggregate report.
```

Comment:
Why have this as a separate script?

I was thinking that we could add the `.csv` code to `common/project_summary.py` and have it execute [when the report is generated](https://github.com/NationalGenomicsInfrastructure/ngi_reports/blob/master/ngi_reports/common/project_summary.py#L35-L36)?

I like the idea of having a generalised function to write a python dict to a file though. Maybe make it part of the `BaseReport` class in [`common/__init__.py`](https://github.com/NationalGenomicsInfrastructure/ngi_reports/blob/master/ngi_reports/common/__init__.py), so that it's available to all report class objects..?